examples/multiphysics/rayleigh-plateau-instability/rayleigh-plateau-J1-We020-Oh0_10_delta0_20/rayleigh-plateau-J1-We020-Oh0_10_delta0_20.prm

```diff
@@ -149,14 +149,24 @@ end
 #---------------------------------------------------
 
 subsection boundary conditions VOF
-  set number = 4
+  set number = 3
   subsection bc 0
     set id   = 0
     set type = dirichlet
     subsection dirichlet
       set Function expression = if(y^2 <= 1.3110e-6, 1, 0)
     end
   end
+  subsection bc 1
+    set id                 = 2
+    set type               = periodic
+    set periodic_id        = 3
+    set periodic_direction = 1
+  end
+  subsection bc 2
+    set id   = 1
+    set type = none
+  end
```

Comment:
I'm not sure if this should be periodic, but I'm not sure which bc it should be... Maybe @AmishgaAlphonius would know?